VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/private-consultation-room-controls/private-consultation-room-controls.component.scss

```diff
@@ -137,49 +138,50 @@
 .dropdown {
   position: relative;
   display: none;
-  @media only screen and (min-width: 320px) and (max-width: 768px) {
+  @media only screen and (max-width: $landscape-large) {
     display: inline-block;
   }
 }
 
 .dropdown-content {
   display: block;
   position: absolute;
-  right: -73px;
+  right: -6.25rem;
   left: auto;
   background-color: govuk-colour('white');
   color: black;
-  font-size: 16px;
-  min-width: 300px;
+  font-size: rem(16px);
+  width: 12.5rem;
   box-shadow: 0px 8px 16px 0px rgba(0, 0, 0, 0.2);
   padding: 12px 16px;
   z-index: 999;
 }
+
 .context-item {
   font-family: 'GDS Transport', arial, sans-serif;
   -webkit-font-smoothing: antialiased;
   margin-left: 4px;
 }
 .portrait {
   display: block;
-  @media only screen and (orientation: landscape) {
+  @media only screen and (orientation: landscape) and (min-width: $breakpoint-small) {
     display: none;
   }
 }
 
 .landscape {
   display: inline-flex;
-  @media only screen and (orientation: portrait) {
+  @media only screen and (orientation: portrait), (max-width: $breakpoint-small) {
     display: none;
   }
-  @media only screen and (min-width: 768px) {
+  @media only screen and (min-width: $landscape-large) {
     display: none;
   }
 }
```

Comment:
I get that this was already here but I'm not a fan of using orientation for this. Any chance these could be changed to be just based on width?